evap/staff/templates/staff_semester_view.html

```diff
@@ -342,9 +342,9 @@ <h3>
 {% block modals %}
     {{ block.super }}
     {% trans 'Delete semester' as title %}
-    {% trans 'Do you really want to delete the semester <strong id="data-label"></strong>?' as question %}
+    {% trans 'Do you really want to delete the semester <strong id="data-label"></strong>? All courses and evaluations will be deleted as well as all results. If you are sure, enter the name of the semester below.' as question %}
     {% trans 'Delete semester' as action_text %}
-    {% include 'confirmation_modal.html' with modal_id='deleteSemesterModal' title=title question=question action_text=action_text btn_type='danger' %}
+    {% include 'confirmation_text_modal.html' with modal_id='deleteSemesterModal' title=title question=question action_text=action_text btn_type='danger' %}
     <script type="text/javascript">
         function deleteSemesterModalAction(dataId) {
             $.ajax({
```

Comment:
The problem with this ajax thing is that the user will not see that anything is "working" until the request gets completed. I just tested and was quite confused until the redirect happened. Anything we can do about it? Maybe make it a form that is submitted instead of an ajax request?

Reply:
we could make it behave like the contact modal where the action button is disabled and says "sending". i thought about that but didn't really want to add more code for something that barely (if ever) happens. would be a more complete version, though.

Reply:
Changing the text to "Sending..." and back shouldn't be that much code, I'd definitely vote in favor of doing that.